src/mgos_barometer_bme280.c

```diff
@@ -155,6 +194,22 @@ bool mgos_barometer_bme280_read(struct mgos_barometer *dev) {
   }
   dev->pressure = (float)P;
 
+// Compensatoin for humidity
+  var1 = ((double)t_fine) - 76800.0;
```

Comment:
Can you execute this block only if the capability is set.

Reply:
Done